src/types/dashboard-types.ts

```diff
@@ -0,0 +1,14 @@
+import { DASHBOARD_HEADER_DATA, DASHBOARD_LIST_DATA } from '../constants/dashboard-constants'
+
+export interface IDashboardDataType {
+  id: number
+  title: string
+  startingTime: string
+  endingTime: string
+  isCompleted : string
+  isFav : string
```

Comment:
isFav should be boolean instead of string

Reply:
it's because this field is of type string in API